istio-control/istio-discovery/metadata-exchange-v2.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: networking.istio.io/v1alpha3
 kind: EnvoyFilter
 metadata:
   name: metadata-exchange
+  namespace: {{ .Release.Namespace }}
```

Comment:
Is this the config namespace? We want these to apply to all namespaces.

Reply:
should it be in `global.configNamespace`?